### PR TITLE
Updating partition_disk automation-tools command to use Ansible

### PR DESCRIPTION
**Pull request by chbrown13**
First attempt showing how to use Ansible with automation-tools

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -90,3 +90,9 @@
     run_command,
     update_packages,
 )
+from fabric.api import env
+
+hosts = env['hosts']
+with open("ansible_runner/inventory/hosts", "w") as file:
+    for host in hosts:
+        file.write(host + '\n')
```
Comment (JacobCallahan) [inline fabfile.py]: do we need a method to remove the added hosts from this file?

**Comment by chbrown13** [comment]
Output for regular disk:
$ fab -D -H hp-bl420cgen8-01.rhts.eng.bos.redhat.com partition_disk
[hp-bl420cgen8-01.rhts.eng.bos.redhat.com] Executing task 'partition_disk'
Not loading passwords
Not loading environment vars
Not loading extra vars
Not loading settings
Not loading ssh key

PLAY [all] *********************************************************************

TASK [Gathering Facts] *********************************************************
ok: [hp-bl420cgen8-01.rhts.eng.bos.redhat.com]

TASK [partition-disk] **********************************************************
changed: [hp-bl420cgen8-01.rhts.eng.bos.redhat.com]

TASK [debug] *******************************************************************
ok: [hp-bl420cgen8-01.rhts.eng.bos.redhat.com] => {
    "msg": "/home"
}

TASK [shell] *******************************************************************
changed: [hp-bl420cgen8-01.rhts.eng.bos.redhat.com]

PLAY RECAP *********************************************************************
hp-bl420cgen8-01.rhts.eng.bos.redhat.com : ok=4    changed=2    unreachable=0    failed=0   


Done.

**Comment by chbrown13** [comment]
Output for resized disk:
$ fab -D -H hp-bl420cgen8-01.rhts.eng.bos.redhat.com partition_disk
[hp-bl420cgen8-01.rhts.eng.bos.redhat.com] Executing task 'partition_disk'
Not loading passwords
Not loading environment vars
Not loading extra vars
Not loading settings
Not loading ssh key

PLAY [all] *********************************************************************

TASK [Gathering Facts] *********************************************************
ok: [hp-bl420cgen8-01.rhts.eng.bos.redhat.com]

TASK [partition-disk] **********************************************************
changed: [hp-bl420cgen8-01.rhts.eng.bos.redhat.com]

TASK [debug] *******************************************************************
ok: [hp-bl420cgen8-01.rhts.eng.bos.redhat.com] => {
    "msg": "/"
}

TASK [shell] *******************************************************************
skipping: [hp-bl420cgen8-01.rhts.eng.bos.redhat.com]

PLAY RECAP *********************************************************************
hp-bl420cgen8-01.rhts.eng.bos.redhat.com : ok=3    changed=1    unreachable=0    failed=0   


Done.

File: fabfile.py (hunk 1 above)
Comment (chbrown13) [reply fabfile.py]: Probably, at the moment it overwrites the file each time but clearing it each time is probably better

File: automation_tools/utils.py [hunk 2]
```diff
@@ -11,6 +11,17 @@
 from six.moves.urllib.request import urlopen
 
 
+def ansible_setup():
+    with open("ansible_runner/inventory/hosts", "w") as file:
+        for host in env['hosts']:
+            file.write(host + '\n')
+
+
+def ansible_cleanup():
```
Comment (chbrown13) [inline automation_tools/utils.py]: @JacobCallahan moved writing hosts to inventory file and added removing hosts to here.

**Comment by kbidarkar** [comment]
I am not implying avoid the move to Ansible but would say, we need to discuss this out first.

Whether we want to convert existing robottelo-ci stuff to Ansible or move to a new project with using ansible stuff. Also one of the main reasons people may want to move to ansible is, it would be run idependent of Jenkins by using `vars` and also to void the requirement to initialize the numerous ENV_VARs.  This would be a major reason to move to Ansible and not because we just need to move to Ansible. :) So, we may have to re-structure things for this purpose, else we may be moving to Ansible but may not solve the actual puprose.

Some time back, while discussing regarding the infra lab stuff, we discussed this and some of the suggestions was to move to an Internal gitlab repo, for the sake of simplicity, along with other Infra stuff.


Also, Jan Hutar started  a discussion on mojo, we can have a discussion on the mojo /email about this topic, let me loop in everyone to that, so that we can discuss and decide about it.

**Comment by kbidarkar** [comment]
I feel we leave this as it is and start with a newer Ansible focused project?

**Comment by kbidarkar** [comment]
As per the above comments, will be closing this PR.

**Comment by lpramuk** [comment]
Rather than fabric utilizing ansible modules I would like to see it opposite way - ansible utilizing fabric

**Pull request closed without merging**